package.json

```diff
@@ -27,6 +27,8 @@
     "static-analysis": "npm run eslint",
     "lint:eslint": "eslint --ignore-path .gitignore $(npm run --silent lint:eslint:ci-reporter)",
     "eslint": "eslint ./",
+    "es5-check:all": "es-check es5 packages/node_modules/@ciscospark/**/dist/*.js",
+    "es5-check:esm": "es-check es5 packages/node_modules/@ciscospark/**/esm/*.js",
```

Comment:
This needs to be updated to reflect that esm files are being built in the `/es/` folder. 

Also, when run, it is failing:

>ES-Check Error:
          ----
          · erroring file: packages/node_modules/@ciscospark/widget-space/es/index.js
          · error: SyntaxError: The keyword 'import' is reserved (1:0)
          · see the printed err.stack below for context
          ----

Looks like it supports a `--module` flag to allow `import` statements, but it doesn't work with the `es5` flag... I'm ok with changing this to:

`es-check --module es6 packages/node_modules/@ciscospark/**/es/*.js`